CalculatorKotlin/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+    kotlin("jvm") version "2.0.21"
+}
```

Comment:
Please add the `application` plugin & set the main class, so `gradle run` will work, and suppress the warning about unsafe access:

```
tasks.named<JavaExec>("run") {
    jvmArgs("--enable-native-access=ALL-UNNAMED")
}

application {
    mainClass.set("org.poach3r.MainKt")
}
```